public/js/actions/breakpoints.js

```diff
@@ -22,15 +23,51 @@ function enableBreakpoint(location: Location) {
   return addBreakpoint(location);
 }
 
+function isOriginalLocation(state, location: Location) {
```

Comment:
sure. I might move those into a utility, but i don't want to go back to the situation where the utility requires selectors and there's a circular dependency